app/code/BVMyBlog/Blog/Block/Post.php

```diff
@@ -0,0 +1,13 @@
+<?php
+declare(strict_types=1);
+namespace BVMyBlog\Blog\Block;
+
+/**
+ * Class Post
+ *
+ * Short description
+ */
+class Post extends \Magento\Framework\View\Element\Template
```

Comment:
What is the usage of current class? If you added phtml template with this block it isn't necessary. You can just use \Magento\Framework\View\Element\Template and assign template to it in layout.xml